src/transaction/validation.rs

```diff
@@ -65,7 +88,22 @@ impl Output {
 }
 
 impl Transaction {
+    #[cfg(feature = "std")]
     pub fn validate(&self, block_height: Word) -> Result<(), ValidationError> {
+        self._validate(block_height, true)
+    }
+
+    #[cfg(feature = "std")]
```

Comment:
Seems like we should be able to validate without `std` if we aren't checking signatures?

Reply:
Fixed in https://github.com/FuelLabs/fuel-tx/pull/57/commits/a77f4e84eb40e75a806e03c41ce44285b6a18bdc